### PR TITLE
Regression - breaking markup on livejournal.com (fix #548)

### LINKED ISSUE #548: Regression - breaking markup on livejournal.com
Regression after "Multiple document.write works incorrectly" fix

### PR DESCRIPTION
**Pull request by LavrovArtem**
\сс @churkin @kirovboris

### CODE REVIEW

File: src/client/sandbox/node/document/writer.js [hunk 1]
```diff
@@ -138,8 +138,12 @@ export default class DocumentWriter {
         var elWithContent = beginMarker;
 
         if (!this.isNonClocedComment) {
-            elWithContent.textContent = this.storedContent + elWithContent.textContent.replace(BEGIN_REMOVE_RE, '');
-            this.storedContent      = '';
+            if (isScriptElement(elWithContent) || isStyleElement(elWithContent)) {
```
Comment (miherlosev) [inline src/client/sandbox/node/document/writer.js]: Typo in `isNonClocedComment`.
Should be `isNonClosedComment`.

Comment (LavrovArtem) [inline src/client/sandbox/node/document/writer.js]: thanks )

**Comment by LavrovArtem** [comment]
FPR

**Comment by kirovboris** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**